File: benchmarks/programs/gcdsum/gcdsum.js
const n = 100;
let s = 0;
for (let i = 1; i <= n; i++) {
  for (let j = 1; j <= n; j++) {
    let a = i, b = j;
    while (b !== 0) {
      const temp = b;
      b = a % b;
      a = temp;
    }
    s += a;
  }
}
console.log(s);

File: benchmarks/programs/gcdsum/gcdsum.rs
fn main() {
    let n: i64 = 100;
    let mut s: i64 = 0;
    for i in 1..=n {
        for j in 1..=n {
            let (mut a, mut b) = (i, j);
            while b != 0 {
                let temp = b;
                b = a % b;
                a = temp;
            }
            s += a;
        }
    }
    println!("{}", s);
}
